changelogs/fragments/1478-filesystem-fix-1457-resizefs-idempotency.yml

```
@@ -0,0 +1,5 @@
+---
+bugfixes:
+  - filesystem - do not fail when ``resizefs=yes`` and ``fstype=xfs`` if there is nothing to do, even if
+    the filesystem is not mounted (https://github.com/ansible-collections/community.general/issues/1457).
+    This only covers systems supporting access to unmounted XFS filesystems. Others will still fail.
```

```suggestion
    the filesystem is not mounted. This only covers systems supporting access to unmounted XFS filesystems.
    Others will still fail (https://github.com/ansible-collections/community.general/issues/1457).
```
In almost all fragments, the link is at the end of the fragment. So let's do it here as well for consistency :)